app/services/policies/policies_list_service.rb

```diff
@@ -10,26 +10,49 @@ class Policies::PoliciesListService
 
   class PoliciesListServiceError < RuntimeError; end
 
-  def self.call(account, builtin: true)
+  def self.apicast_registry_url
+    ThreeScale.config.sandbox_proxy.apicast_registry_url
+  end
+
+  delegate :apicast_registry_url, to: 'self.class'
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(account, builtin: true)
+    @account = account
+    @builtin = builtin
+  end
+
+  attr_reader :account
+
+  def builtin?
+    !!@builtin
```

Comment:
Another option is in the line 25 doing `@builtin = !!builtin` and here just `@builtin` so we don't have to do convert through `!!` for every request to `builtin?` 😄

Reply:
In fact, we don't even need to convert it if we don't want to. `builtin?` is only meant for readability. This works just the same:

```ruby
def builtin?
  @builtin
end

Or even this:

```ruby
attr_reader :builtin
alias builtin? builtin
```

Reply:
Yup, also good 👍 😎

Reply:
I like the latter the most

Reply:
Done